go/libraries/doltcore/doltdb/doltdb.go

```diff
@@ -1184,7 +1184,7 @@ func (ddb *DoltDB) deleteRef(ctx context.Context, dref ref.DoltRef) error {
 		}
 	}
 
-	_, err = ddb.db.Delete(ctx, ds)
+	_, err = ddb.db.withReplicationStatusController(replicationStatus).Delete(ctx, ds)
```

Comment:
Occurs to me that this is the better overall solution to use everywhere (rather than params in affected methods), but can be punted on for now I guess